crates/sui-types/src/transaction.rs

```diff
@@ -915,6 +915,14 @@ impl ProgrammableTransaction {
                 value: config.max_programmable_tx_commands().to_string()
             }
         );
```

Comment:
Would it be too hard to also check for the number of dependency packages here?

Reply:
How is the check performed right now? I can move the check here.